crates/api_models/src/payments.rs

```diff
@@ -616,6 +616,9 @@ pub struct Card {
     pub nick_name: Option<Secret<String>>,
 }
 
+#[derive(Eq, PartialEq, Clone, Debug, serde::Deserialize, serde::Serialize, ToSchema)]
+pub struct CardRedirectData {}
```

Comment:
Let's have CardRedirectData as enum with variants for KNET, BENEFIT. So that in future if any PM requires input, changes won't impact the api contract of other PM's